comixed-frontend/src/app/user/adaptors/authentication.adaptor.ts

```diff
@@ -137,6 +137,11 @@ export class AuthenticationAdaptor {
     this.store.dispatch(new AuthLogout());
   }
 
+  authenticationFailed(): void {
+    this.store.dispatch(new AuthLoginFailed());
+    this.store.dispatch(new AuthHideLogin());
```

Comment:
I don't think, in the case of a failed login attempt, we should hide the login dialog. We should only hide the login dialog if the user explicitly clicks **Cancel** on it.

Reply:
Ok.